server/controllers/api/getlist.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function(app) {
+	
+	var bodyParser = require('body-parser')
+	var jsonParser = bodyParser.json()
+	var urlencodedParser = bodyParser.urlencoded({ extended: false })
+
+	app.use(bodyParser.urlencoded({ extended: false }))
+	app.use(bodyParser.json())
+	
+	app.post('/api/getlist', function (req, res) {
+		console.log('inside post');
+  		if (!req.body) return res.sendStatus(400);
+
+  		var requested = req.body;
+
+  		res.header("Access-Control-Allow-Origin", "*");
+
+		app.db.select([
```

Comment:
Hmmmm... Idealy on requesting details of a tree we should return picture of it as well. I do suppose, sending full colunms from the table will not impact traffic too much, but we can discuss. Anyway as a part of #6 we should have a mechinism of getting picture in addition.